README.md

```diff
@@ -1,4 +1,63 @@
-# Logstash Plugin
+# Logstash Output Plugin
+
+---
+
+This plugin is now in maintenance mode. This is because the OpenSearch Project created a new Logstash output plugin 
+[logstash-output-opensearch](https://github.com/opensearch-project/logstash-output-opensearch) which ships events from 
+Logstash to OpenSearch 1.x and Elasticsearch 7.x clusters, and also supports SigV4 signing. Having similar functionality
+plugins can be redundant, so we plan to eventually replace this logstash-output-amazon_es plugin with the logstash-output-opensearch
+plugin.
+
+To help you migrate to [logstash-output-opensearch](https://github.com/opensearch-project/logstash-output-opensearch) plugin, please 
+find below a brief migration guide.
+
+## Migrating to logstash-output-opensearch plugin
+
+---
```

Comment:
I don't think the document needs this extra line. The "##" header already provides one.

Reply:
I'll remove it, it didn't show up in my IntelliJ Markdown Preview so I had added it.